src/main/java/org/skriptlang/skript/bukkit/spark/SparkUtils.java

```diff
@@ -0,0 +1,162 @@
+package org.skriptlang.skript.bukkit.spark;
+
+import me.lucko.spark.api.Spark;
+import me.lucko.spark.api.SparkProvider;
+import me.lucko.spark.api.statistic.StatisticWindow.CpuUsage;
+import me.lucko.spark.api.statistic.StatisticWindow.MillisPerTick;
+import me.lucko.spark.api.statistic.StatisticWindow.TicksPerSecond;
+import me.lucko.spark.api.statistic.misc.DoubleAverageInfo;
+import me.lucko.spark.api.statistic.types.DoubleStatistic;
+import me.lucko.spark.api.statistic.types.GenericStatistic;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.UnknownNullability;
+
+import java.util.Arrays;
+
+/**
+ * Utility class for accessing Spark's API.
+ */
+public class SparkUtils {
+
+	private static volatile DoubleStatistic<CpuUsage> cpuProcess;
+	private static volatile DoubleStatistic<CpuUsage> cpuSystem;
+	private static volatile GenericStatistic<DoubleAverageInfo, MillisPerTick> mspt;
+	private static volatile DoubleStatistic<TicksPerSecond> tps;
+
+	@Contract(pure = true)
+	private static @NotNull Spark spark() {
+		return SparkProvider.get();
+	}
+
+	/**
+	 * Checks for MSPT support. This should be checked before using the MSPT methods.
+	 * @return whether MSPT statistics are supported.
+	 */
+	public static boolean supportsMspt() {
+		return mspt() != null;
```

Comment:
How does checking if it's null check if it's supported?

Reply:
It can be disabled by using the `/spark tickmonitor` command.

Reply:
That wasn't exactly my question. Also, this brings up additional concern. This check would not work in this scenario:
1. Start server (tickmonitor enabled)
2. Do something that causes this lazy field to be computed
3. Disable tickmonitor
4. This check is now inaccurate